Documentation/compatibility/uri-unicode-bidirectional-characters.md

```diff
@@ -0,0 +1,38 @@
+## Allow Unicode Bidirectional Control Characters in URIs
+
+### Scope
+Minor
+
+### Version Introduced
+4.7.2
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+Unicode specifies several special control characters used to specify the orientation of text.
+In previous versions of the .NET Framework, these characters would be incorrectly stripped
+from all URIs. The characters would be stripped even if they were present in their percent
+encoded form. In order to better follow [RFC 3987](http://tools.ietf.org/html/rfc3987), we now allow these characters in URIs.
+When found un-encoded in a URI they will be percent encoded. When found percent encoded they
```

Comment:
un-encoded --> unencoded
comma after URI
will be --> are (it's preferable to use present rather than future tense) 
2 occurrences of percent encoded --> percent-encoded